consensus/broadcast/delayedBroadcast_test.go

```diff
@@ -299,6 +315,83 @@ func TestDelayedBlockBroadcaster_HeaderReceivedForNotRegisteredDelayedDataShould
 	assert.False(t, txBroadcastCalled.IsSet())
 }
 
+func TestDelayedBlockBroadcaster_HeaderReceivedWithoutSignaturesForShardShouldNotBroadcastTheData(t *testing.T) {
+	var logOutput bytes.Buffer
+	var logMutex sync.Mutex
+
+	customFormatter := &logger.PlainFormatter{}
+	observer := syncLogObserver{
+		buffer:    &logOutput,
+		mutex:     &logMutex,
+		formatter: customFormatter,
+	}
+	err := logger.AddLogObserver(&observer, customFormatter)
+	require.Nil(t, err)
+
+	originalLogPattern := logger.GetLogLevelPattern()
+	err = logger.SetLogLevel("*:TRACE")
+	require.Nil(t, err)
```

Comment:
```go
defer func() {
	err = logger.RemoveLogObserver(&observer)
	require.Nil(t, err)
	err = logger.SetLogLevel(originalLogPattern)
	require.Nil(t, err)
}()
```
after this line, instead of calling them at the end.. valid for all tests